src/python/pants/backend/docker/util_rules/dependencies.py

```diff
@@ -33,7 +34,12 @@ async def inject_docker_dependencies(
             UnparsedAddressInputs(
                 (v for v in dockerfile_info.from_image_build_args.to_dict().values() if v),
                 owning_address=dockerfile_info.address,
-                description_of_origin="TODO(#14468)",
+                description_of_origin=softwrap(
+                    f"""
+                    the FROM arguments from the Dockerfile {dockerfile_info.source}
```

Comment:
This will not read well. 

"... from the Dockerfile Dockerfile ..." will not be uncommon.